straxen/plugins/event_processing.py

```diff
@@ -681,6 +681,16 @@ class CorrectedAreas(strax.Plugin):
         default='cmt://relative_light_yield?version=ONLINE&run_id=plugin.run_id',
         help='Relative light yield (allows for time dependence)'
     )
+    
+    region_linear = straxen.URLConfig(
+        default=28,
+        help='linear cut for ab region'
+    )
+    
+    region_circular = straxen.URLConfig(
+        default=60,
+        help='circular cut for ab region'
+    )
```

Comment:
@Jianyu010 can you explain a bit more in the help region? Maybe linking a note here would be usefull

Reply:
Linked the note